presentation/chapters/de-DE/error-handling.chapter

```diff
@@ -0,0 +1,127 @@
+# Fehlerbehandlung
+
+---
+
+Die Fehlerbehandlung ist explizit in Rust.
```

Comment:
Wie im Englischen: verzichte auf das Pronomen. Danach ist der Satzbau ungewohnt.

"Fehlerbehandlung in Rust ist explizit."